candy-machine/program/src/lib.rs

```diff
@@ -395,9 +430,6 @@ pub mod candy_machine {
             ],
             &[&authority_seeds],
         )?;
-        let instruction_sysvar_account_info = instruction_sysvar_account.to_account_info();
-
-        let instruction_sysvar = instruction_sysvar_account_info.data.borrow();
```

Comment:
I can't comment this line in the PR:
```
 let associated_token =
            Pubkey::from_str("ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL").unwrap();
```
but we should fix this as well, as complete waste of compute units to perform a String to Pubkey conversion on chain.  Instead the conversion can be performed at build time with a:
```
const ASSOCIATED_TOKEN: Pubkey = solana_program::pubkey!("ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL");
```